### PR TITLE
#2 Recognize text

### LINKED ISSUE #2: Recognize text
As a user after pointing at some text I'd like for it to be recognized and presented to me as an overlay on a preview.

Dev notes:
- use ML Kit Vision for Firebase
- text should be black with semitransparent white background
- preferably is for recognized text to overlap original text on the preview. This could be omitted if not possible.

### PR DESCRIPTION
**Pull request by daniellojewski**
Closes #2

### CODE REVIEW

File: lib/text_recognition/preview/widget.dart [hunk 1]
```diff
@@ -52,17 +56,19 @@ class _CameraPreviewWidgetState extends State<CameraPreviewWidget> {
...
       ),
     );
   }
 
-  Widget _buildCameraPreview(BuildContext context, CameraController controller) {
+  Widget _buildCameraPreview(BuildContext context, ReadyPreviewState state) {
+    final controller = state.controller;
     final size = MediaQuery.of(context).size;
-    final deviceRatio = size.width / size.height;
+    double deviceRatio = size.width / size.height;
```
Comment (daniellojewski) [inline lib/text_recognition/preview/widget.dart]: `double` - > `final`

File: lib/text_recognition/preview/widget.dart [hunk 2]
```diff
@@ -52,17 +56,19 @@ class _CameraPreviewWidgetState extends State<CameraPreviewWidget> {
...
+
```
Comment (daniellojewski) [inline lib/text_recognition/preview/widget.dart]: Redundant empty line

File: android/app/google-services.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{
+  "project_info": {
```
Comment (bkraszewski) [inline android/app/google-services.json]: I think this file should be excluded from GIT, someone may steal your account and charge your card :)

**Review by bkraszewski — CHANGES_REQUESTED** [review]

File: android/app/google-services.json (hunk 3 above)
Comment (daniellojewski) [reply android/app/google-services.json]: My bad, removed

**Pull request merged**